apps/meteor/server/models/raw/Subscriptions.ts

```diff
@@ -706,13 +706,15 @@ export class SubscriptionsRaw extends BaseRaw<ISubscription> implements ISubscri
 		return this.updateOne(query, update);
 	}
 
-	updateHideUnreadStatusById(_id: string, hideUnreadStatus: true): Promise<UpdateResult> {
+	updateHideUnreadStatusById(_id: string, hideUnreadStatus: boolean): Promise<UpdateResult> {
 		const query = {
 			_id,
 		};
 
 		const update: UpdateFilter<ISubscription> = {
 			$set: {
+				// @ts-expect-error - Check types of subscription
```

Comment:
👀

Reply:
Yep. Basically:
ISubscription.hideUnreadStatus has "true" as type
But, when an endpoint calls updateHideUnreadStatusById, it passes a boolean 🤔 (x === '1')

So, not sure which of the types should prevail. IMHO, ISubscription should have boolean as type instead of true.

Reply:
why not 

```
const update: UpdateFilter<ISubscription> = hideUnreadStatus
			? {
					$set: {
						hideUnreadStatus,
					},
			  }
			: {
					$unset: {
						hideUnreadStatus: 1,
					},
			  };
```

Reply:
It changes what the function was doing originally, but works for the type. Since it doesn't change the logic, let me do it